src/util/sock.h

```diff
@@ -10,6 +10,7 @@
 #include <util/time.h>
 
 #include <chrono>
+#include <memory>
```

Comment:
11e59842e30ab5 nit if you retouch, add this header to the .cpp file as well

Reply:
Added.